backend/src/components/lookup.js

```diff
@@ -55,31 +55,32 @@ const fundingModelType = [
   },
 ];
 
-function parseProgramYear(value) {
+function parseProgramYear(programYearList) {
   const programYears = {
-    current: undefined,
```

Comment:
It technically doesn't matter but we are still using current. So renewal is defined but not current.

On further thought, do current and renewal even need to be part of programYears. It seems they are maybe just local variables used to determine newApp? There's no point in sending in including it if it's not used in the front end.

Reply:
We shouldn't be using current anymore. I'm not sure what happened to that change - but I guess it didn't make it here? Anyways... it's updated now

Reply:
Is renewal used in the frontend? Just wondering if it needs to be part of programYears.

Reply:
On closer look it is used in two files.